runtime/src/installed_scheduler_pool.rs

```diff
@@ -519,15 +526,15 @@ impl BankWithSchedulerInner {
                 );
                 Err(SchedulerAborted)
             }
-            SchedulerStatus::Stale(_pool, _result_with_timings) => {
+            SchedulerStatus::Stale(pool, _result_with_timings) => {
+                let pool = pool.clone();
```

Comment:
because this `pool` is borrowed and we need to re-lock `scheduler` with the write access:

```
error[E0505]: cannot move out of `scheduler` because it is borrowed
   --> runtime/src/installed_scheduler_pool.rs:532:22
    |
517 |         let scheduler = self.scheduler.read().unwrap();
    |             --------- binding `scheduler` declared here
518 |         match &*scheduler {
    |                 --------- borrow of `scheduler` occurs here
...
532 |                 drop(scheduler);
    |                      ^^^^^^^^^ move out of `scheduler` occurs here
...
552 |                 pool.register_timeout_listener(self.do_create_timeout_listener());
    |                 ---- borrow later used here

For more information about this error, try `rustc --explain E0505`.
error: could not compile `solana-runtime` (lib) due to 1 previous error

```